bootstrap.d/app-editors.yml

```diff
@@ -1,4 +1,33 @@
 packages:
+  - name: mg
+    source:
+      subdir: 'ports'
+      git: 'https://github.com/troglobit/mg.git'
+      tag: 'v3.4'
+      version: '3.4'
+      tools_required:
+        - host-autoconf-v2.69
+        - host-automake-v1.15
+        - host-pkg-config
+      regenerate:
+        - args: ['./autogen.sh']
+    tools_required:
+      - host-pkg-config
```

Comment:
Testing suggests that `host-pkg-config` is not required as a tool during configure.